### PR TITLE
Cargo - Add item index and damage to cargo menu

### PR DESCRIPTION
**Pull request by LinkIsGrim**
**When merged this pull request will:**
- Title, see #9643 

### IMPORTANT

- If the contribution affects [the documentation](https://github.com/acemod/ACE3/tree/master/docs), please include your changes in this pull request so the documentation will appear on the [website](https://ace3.acemod.org/).
- [Development Guidelines](https://ace3.acemod.org/wiki/development/) are read, understood and applied.
- Title of this PR uses our standard template `Component - Add|Fix|Improve|Change|Make|Remove {changes}`.

### CODE REVIEW

**Comment by johnb432** [comment]
Do we want to display health or damage?
Imo health is more intuitive.

**Comment by rautamiekka** [comment]
> Do we want to display health or damage? Imo health is more intuitive.

Agree.

**Comment by LinkIsGrim** [comment]
Health doesn't have good translations, and not really applicable since we're talking about inanimate objects.

**Comment by rautamiekka** [comment]
> Health doesn't have good translations

Just use `(30%)`.

> , and not really applicable since we're talking about inanimate objects.

That's just the other side of the coin, 30% damage being 70% health left, and like we both said, unintuitive.

File: addons/cargo/functions/fnc_onMenuOpen.sqf [hunk 1]
```diff
@@ -64,19 +64,24 @@ if (GVAR(interactionParadrop)) then {
...
     {
         _displayName = [_x, true] call FUNC(getNameItem);
         _itemSize = _x call FUNC(getSizeItem);
+        _damage = if (_x isEqualType "") then {0} else {damage _x};
+        _damageStr = ((_damage * 100) toFixed 0) + "%";
 
         if (_itemSize >= 0) then {
             _index = if (GVAR(interactionParadrop)) then {
-                _ctrl lbAdd format ["%1 (%2s)", _displayName, GVAR(paradropTimeCoefficent) * _itemSize]
+                _ctrl lbAdd format ["%1. %2 (%3s)", _forEachIndex + 1, _displayName, GVAR(paradropTimeCoefficent) * _itemSize];
```
Comment (johnb432) [inline addons/cargo/functions/fnc_onMenuOpen.sqf:78]:
```suggestion
                _ctrl lbAdd format ["%1. %2 (%3s)", _forEachIndex + 1, _displayName, GVAR(paradropTimeCoefficent) * _itemSize]
```

File: addons/cargo/functions/fnc_onMenuOpen.sqf [hunk 2]
```diff
@@ -64,19 +64,24 @@ if (GVAR(interactionParadrop)) then {
...
             } else {
-                _ctrl lbAdd _displayName
+                _ctrl lbAdd format ["%1. %2", _forEachIndex + 1, _displayName];
```
Comment (johnb432) [inline addons/cargo/functions/fnc_onMenuOpen.sqf:80]:
```suggestion
                _ctrl lbAdd format ["%1. %2", _forEachIndex + 1, _displayName]
```

**Review by johnb432 — APPROVED** [review]
Remove the semicolons and I'm happy.

**Pull request merged**